pycbc/events/stat.py

```diff
@@ -1174,26 +1182,41 @@ def coinc_multiifo(self, s, slide, step, to_shift,
         benchmark_logvol = s[0][1]['benchmark_logvol']
         network_logvol -= benchmark_logvol
 
+        # Use prior histogram to get log signal rate
         stat = {ifo: st for ifo, st in s}
         logr_s = self.logsignalrate_multiifo(stat,
                                              slide * step, to_shift)
 
-        loglr = logr_s + network_logvol - ln_noise_rate
+        # Calculate noise rate in each bin given uniform phase time diffs
```

Comment:
I'm not seeing why we want to calculate noise rate in each bin if the signal rate term is now a proper density, i.e. rate per volume not rate per bin